### PR TITLE
views

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
Create user card partial
Render user collection
Render views
Add basic styling

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
```
Comment (DanSam5K) [inline app/models/user.rb:5]: - [ ] Please kindly add association between the user and post also, for example, the user should also have many post associated with the user foreign key

**Review by DanSam5K — CHANGES_REQUESTED** [review]
# Hi @yahyaelganyni1, ♻️Required Changes ♻️
## Highlights 💥 💥 
- [x] Good job on making sure all linter checks passed ✔️ 
- [x] Excellent job with implementing the project as instructed ✔️ 

Good job so far!
There are a few issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @yahyaelganyni1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (yahyaelganyni1) [reply app/models/user.rb:5]: hello @DanSam5K 
      I think you are in the wrong PR this is for the view, not the association ^^
and this is my second time doing this and the previous reviewers accepted the code so I don't understand what is the problem?

[the previous PR](https://github.com/yahyaelganyni1/the-daily-blog/pull/4)

Comment (DanSam5K) [reply app/models/user.rb:5]: Hi @yahyaelganyni1,
I checked and the files present for me to review was just a single file which did not reflect your work and I saw you had two PR, with the task contain in the other PR so I reviewed the work on it.

**Pull request merged**